src/project/context/flowr-analyzer-functions-context.ts

```diff
@@ -0,0 +1,64 @@
+import { AbstractFlowrAnalyzerContext } from './abstract-flowr-analyzer-context';
+import type { FlowrAnalyzerContext } from './flowr-analyzer-context';
+import {
+	FlowrAnalyzerPackageVersionsPlugin
+} from '../plugins/package-version-plugins/flowr-analyzer-package-versions-plugin';
+
+export enum FunctionTypes {
```

Comment:
it is weird to have a function type `symbol`, maybe rename this to `ExportTypes`